src/ui/connect/connector.ts

```diff
@@ -73,6 +73,10 @@ export default class Connector implements ExtensionActions {
         this.port.postMessage({type: 'mark-news-as-read', data: ids});
     }
 
+    loadConfig(local: boolean) {
```

Comment:
Use destructuring (`options: {local: boolean}`) to make it look clear, otherwise `loadConfig(true)` or `loadConfig(false)` doesn't tell what exactly is changed.